src/CypherNet.Core/GraphStore.cs

```diff
@@ -27,7 +27,7 @@ public class GraphStore
         #region Constructors and Destructors
 
         /// <summary>
-        /// Initialises a new instance of the <see cref="GraphStore"/> class.
+        /// Initialises a new instance of the <see cref="GraphStore"/> class, usable before Neo4J version 2.2.0.
```

Comment:
One very minor comment on this.  This constructor will still be usable with 2.2.0, but the server will have to be configured to not use auth: http://neo4j.com/docs/stable/security-server.html#security-server-auth